File: AspNetMvcUnleashed/0672329980_source/Chapter07/CS/MvcApplication1/MvcApplication1/Controllers/SurveyController.cs
using System.Collections.Generic;
using System.Web.Mvc;

namespace MvcApplication1.Controllers
{
    public class SurveyController : Controller
    {
        public ActionResult Create()
        {
            return View();
        }

        [AcceptVerbs(HttpVerbs.Post)]
        public ActionResult Create(List<string> source)
        {
            return View();
        }

    }
}

File: AspNetMvcUnleashed/0672329980_source/Chapter07/VB/MvcApplication1/MvcApplication1/Controllers/SurveyController.vb
Public Class SurveyController
    Inherits System.Web.Mvc.Controller

    '
    ' GET: /Survey/Create

    Function Create() As ActionResult
        Return View()
    End Function


    ' POST: /Survey/Create
    <AcceptVerbs(HttpVerbs.Post)> _
    Function Create(ByVal source As List(Of String)) As ActionResult
        Return View()
    End Function

End Class
